apiClient/types.ts

```diff
@@ -22,8 +22,8 @@ export type ApiEventMetadata =
 
 export type ApiEvent = {
   id: number
-  created_at: string
-  updated_at: string
+  created_at?: string
+  updated_at?: string
```

Comment:
These can be deleted as the API never sends these down. https://github.com/iron-fish/ironfish-api/blob/master/src/events/utils/event-translator.ts#L10-L18

Reply:
Removed